apps/src/pythonlab/pyodideWebWorker.js

```diff
@@ -73,6 +80,16 @@ self.onmessage = async event => {
   self.postMessage({type: 'run_complete', message: results, id});
 };
 
+async function runInternalCode(code, id) {
```

Comment:
Non-blocking/not attached to this feedback: Would it be worth naming this to indicate it returns a boolean for success or failure? Off-the-top-of-head ideas: `executeCodeAndReturnStatus` or `runCodeAndCheckSuccess` or `handleCodeAndReturnStatus`...

Reply:
I added a clarifying comment that talks about the return value, lmk if that makes sense!